app/processing/statement/components/get-scheme.js

```diff
@@ -0,0 +1,11 @@
+const getScheme = async (paymentRequest) => {
+  return {
+    name: 'Sustainable Farming Incentive',
+    shortName: 'SFI',

```

Comment:
I think this is fine for now.

We probably should have a set of consts for these values and schemes.  The `schemeId` in the payment request should allow us to get these. 

But yeah.  Another day.

Reply:
Abi, Simon and I have acc'ly been working on this exact issue so once we merge in you can take up those constants :D